packages/zero-runtime/src/styled.spec.tsx

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable material-ui/no-empty-box */
+import * as React from 'react';
+import { styled } from '@mui/zero-runtime';
+
+declare module '@mui/zero-runtime' {
+  interface ThemeArgs {
+    theme: {
+      palette: {
+        primary: Record<string, string>;
+      };
+      mixins: {
+        toolbar: Record<string, string>;
+      };
+      typography: {
+        body1: Record<string, string>;
+      };
+    };
+  }
+}
+
+const B = styled.a<{ isRed?: boolean }>({
+  color(props) {
+    return props.isRed ? 'red' : 'blue';
+  },
```

Comment:
```suggestion
  color: (props) => props.isRed ? 'red' : 'blue',
```

This is the syntax most people would use, no? Is the one in the code working? I was surprised honestly :)